components/style/gecko/non_ts_pseudo_class_list.rs

```diff
@@ -46,4 +46,8 @@ pseudo_class_list! {
     ("read-only", ReadOnly, _, IN_READ_WRITE_STATE, _),
 
     ("-moz-browser-frame", MozBrowserFrame, mozBrowserFrame, _, PSEUDO_CLASS_INTERNAL),
+
+    // FIXME(SimonSapin): bz says that this pseudo-class is currently enabled for all stylesheets,
+    // but we could try to restrict it to user-agent stylesheets by adding PSEUDO_CLASS_INTERNAL here.
+    ("-moz-table-border-nonzero", MozTableBorderNonzero, mozTableBorderNonzero, _, _),
```

Comment:
Let's just mark it internal. I've filed [a Gecko bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341925) to address this before stylo gets enabled, and posted [an intent to unship](https://groups.google.com/d/msg/mozilla.dev.platform/jtdC_pUmCew/IQIdkw9CCQAJ) to dev-platform.